### PR TITLE
Clarify the function of the extra integer given by the debugger

### LINKED ISSUE #242: Output prints two extra characters on evry line: ,0
On the output panel of vscode each line ends with `,0` (comma - zero)

image

### PR DESCRIPTION
**Pull request by linkpy**
Fixes #242

### CODE REVIEW

**Review by Calinou — APPROVED** [review]
Thanks!

File: src/debugger/mediator.ts [hunk 1]
```diff
@@ -29,7 +29,13 @@ export class Mediator {
 
 				let lines: string[] = parameters;
 				lines.forEach((line) => {
-					this.output.appendLine(line);
+					let message_content: string = line[0];
+					//let message_kind: number = line[1];
```
Comment (Shatur95) [inline src/debugger/mediator.ts:33]: This PR fixed the issue for me, but why do you left this commented code?

Comment (linkpy) [reply src/debugger/mediator.ts:33]: In case futur contributor wonders why `message_content` is `line[0]`.

Comment (Shatur95) [reply src/debugger/mediator.ts:33]: I understand text comments, but `//let message_kind: number = line[1];` look weird :)

Comment (linkpy) [reply src/debugger/mediator.ts:33]: No i was indeed talking about that comment line. If i dont comment it out there is a warning saying "unused variable".

Comment (Shatur95) [reply src/debugger/mediator.ts:33]: Got it, thanks :)

**Pull request merged**